Detectors/MUON/MCH/Workflow/CMakeLists.txt

```diff
@@ -216,15 +222,15 @@ o2_add_executable(
             src/ClusterTransformerSpec.cxx
             src/DigitFilteringSpec.cxx
             src/DigitReaderSpec.cxx
-            src/TrackAtVertexSpec.cxx
+            src/EventFinderSpec.cxx
```

Comment:
why is the time clustering device dependent on the event finder ?

Reply:
sorry, my previous message didn't show up... it's not the time clustering, it's the reco-workflow ;-)

Reply:
indeed. got confused by the way GitHub diffs are presented. sorry about that.